### PR TITLE
feat : home and products pages

### PR DESCRIPTION
**Pull request by rohit8686**
Live Link : [https://eleco-react.netlify.app/](https://eleco-react.netlify.app/)

Home page contains : 

- Nav bar (has search functionality)
- Categories (redirects to product page by showing products from selected category) 
- New Arrivals

Products Page contains : 

- Filter functionality (sort by price, categories, price range, rating)
- Products

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (romabulani) [inline public/index.html]: You can remove these starter comments

File: src/frontend/pages/Products/components/Filters.jsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+          name="rating"
+          id="above4"
+          className="rating"
+          onChange={() => productDispatch({ type: "RATING", payload: 4 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 4
+              ? true
+              : false
```
Comment (romabulani) [inline src/frontend/pages/Products/components/Filters.jsx]: you can directly write checked= {productState.rating === 4}

File: src/frontend/pages/Products/components/Filters.jsx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+          onChange={() =>
+            productDispatch({ type: "SORT", payload: "LOW_TO_HIGH" })
+          }
+          checked={
+            productState.sort === ""
+              ? false
+              : productState.sort === "LOW_TO_HIGH"
+              ? true
+              : false
+          }
```
Comment (romabulani) [inline src/frontend/pages/Products/components/Filters.jsx]: here also you can write  checked={
            productState.sort === "LOW_TO_HIGH"  }

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```
Comment (romabulani) [inline public/manifest.json:25]: You can remove this file its not needed

**Review by romabulani — APPROVED** [review]
- Good folder structure
- Have given some suggestions, apart from them, PR looks good to merge

File: src/frontend/pages/Products/components/Filters.jsx (hunk 2 above)
Comment (rohit8686) [reply src/frontend/pages/Products/components/Filters.jsx]: yeah thanks @romabulani, will change it

**Review by shrey27 — APPROVED** [review]
Overall code structure looks good
PR is good to merge

**Pull request merged**